requirements.txt

```diff
@@ -0,0 +1,19 @@
+# actual dependencies
+grid2op == 1.9.7
+pandapower == 2.13.1
+networkx
+matplotlib
+l2rpn-baselines == 0.8.0
+stable-baselines3 == 2.1.0
```

Comment:
I really don't like `==` in "actual dependencies" (for tests or CI to fix the version is ok and even a good practice, but otherwise it's often a terrible idea).
Unless, again, you commit to upgrade this package ad vitam eaternam when new versions of grid2op, pandapower, l2rpn-baselines or stable-baselines3 will be updated.

Reply:
There are two files with dependencies: requirements.txt and pyproject.toml. Which one is used for what ?
For the one that is used for building the package by anyone, don't fix the dependencies, use ">=". And yet in the CI test indeed fix the dependencies

Reply:
I changed the requirements.txt file for the CI/CD to run with upgraded version of the packages, and also deleted the non necessary dependencies.
For the .toml, used to package, I think that this is not really useful to allow more then what it is for the moment because I don't think that a lot of person will use our wheel. Furthermore with actual dependencies all the updates will be taken until a new base version is released (from 1.8.6 to 2.0.0 for example).